pkg/usagestats/stats.go

```diff
@@ -340,6 +346,92 @@ func (s *Statistics) Record(v float64) {
 	}
 }
 
+type MultiStatistics struct {
+	values  map[string]*Statistics
+	keyName string
```

Comment:
We can currently place one "key" (label) on MultiStatistics and MultiCounters for simplicity, in the future this can be extended if needed.